cmd/sync/aws.go

```diff
@@ -29,6 +30,24 @@ func NewAWSClient(data []byte) (*AWSClient, error) {
 		return nil, fmt.Errorf("error validating config: %v", err)
 	}
 
+	if cfg.Region == "self" {
+		metaSession, err := session.NewSession()
+		if err != nil {
+			return nil, err
+		}
+		
+		metaClient := ec2metadata.New(metaSession)
+		if metaClient.Available() != true {
```

Comment:
```suggestion
		if !metaClient.Available() {
```
for consistency with the rest of the code